src/temp/AssignedUsers/AssignedUsersList.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { IfPermission } from '@folio/stripes/core';
+import {
+  Row,
+  Col,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import {
+  COLUMN_MAPPING,
+  COLUMN_WIDTH,
+  VISIBLE_COLUMNS,
+} from './constants';
+import AssignMembers from './AssignUsers';
+
+const AssignedUsersList = ({ users, assignUsers, isFetching }) => (
+  <>
+    <Row end="xs">
```

Comment:
it shouldn't be part of `AssignedUsersList`, it should be part of `Accordion`

https://github.com/folio-org/stripes-components/blob/master/lib/Accordion/Accordion.js#L29